resources/js/Components/LanguageSelector.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<div class="mismatchfinder__language-selector">
 		<div class="languageSelector__mobile-header">
-			<span>{{ $i18n( 'language-selector-mobile-header' ) }}</span>
+			<span>{{ $i18n('language-selector-mobile-header') }}</span>
```

Comment:
I know this is not 100% consistent across the project but it seems like here there were spaces around the parenthesis and now in this file it has been changed. We need to add a rule to the linter to make this consistent. It's not that relevant for this particular PR but we need to talk about this :)

Reply:
100% agree! :)